stats/api/views.py

```diff
@@ -8,6 +12,28 @@
 from datetime import datetime, timedelta
 
 
+class HomeView(View):
```

Comment:
This class name isn't very suggestive. Maybe `StatsView` / `ChartsView` would be better suited.